src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.Windows.cs

```diff
@@ -42,5 +41,13 @@ private static unsafe CultureInfo GetUserDefaultUICulture()
 
             return InitializeUserDefaultCulture();
         }
+
+        internal static string? UserDefaultLocaleName { get; private set; } = GlobalizationMode.Invariant ?
```

Comment:
>private [](start = 61, length = 7)

why we have it private set if it is internal property anyway?

Reply:
We should just be setting it from within `CultureInfo` -- I can remove the `private set`.

Reply:
I'll remove it, I just realized I'll have to update the PR, because in `CultureInfo.ClearChache()` I'm not using InvariantCulture.Name when in invariant mode.

Reply:
Done.